app/main.py

```diff
@@ -2,0 +1,35 @@
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None :
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{Name: " + f"{self.name}, Health: " \
+            + f"{self.health}, Hidden: " + str(self.hidden) + "}"
```

Comment:
The `__repr__` method could be improved for better readability. Consider using an f-string for the entire return statement: `return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"`.